stat.go

```diff
@@ -125,6 +125,32 @@ func ChiSquare(obs, exp []float64) float64 {
 	return result
 }
 
+// CircularMean returns the circular mean of the dataset.
```

Comment:
Could you provide the formula like we do in Mean? It makes it clear for those who don't know.